jenkins/test/testsuite/Jenkinsfile

```diff
@@ -59,13 +60,28 @@ pipeline {
                 }
             }
             steps {
-                sh "./test.sh ${JOB_NAME} --s3-bucket ${ARTIFACT_BUCKET_NAME} --opensearch-version ${opensearch_version} --build-id ${build_id} --architecture ${architecture} --test-run-id ${test_run_id}"
+                script {
+                    basePath = "https://ci.opensearch.org/ci/dbc/bundle-build/${opensearch_version}/${build_id}/${platform}/${architecture}"
```

Comment:
Can we make this a parameter to the job? We just had a build break yesterday night due to building these paths inconsistently

Reply:
`https://ci.opensearch.org/ci/dbc/bundle-build/`should be a parameter to the job?

Reply:
Even more - ` basePath = "https://ci.opensearch.org/ci/dbc/bundle-build/${opensearch_version}/${build_id}/${platform}/${architecture}"` could be passed in a parameter, only the architecture is used to determine the agent node type all the rest of the values are passed around to no effect.

...lets come back to this after you merge to build momentum